src/org/exist/backup/SystemExport.java

```diff
@@ -918,8 +919,8 @@ public void startElement( String uri, String localName, String qName, Attributes
                             serializer.endElement( Namespaces.EXIST_NS, "deleted", "deleted" );
                         }
                     }
-                } catch(final PermissionDeniedException pde) {
-                    throw new SAXException("Unable to process :" + qName + ": " + pde.getMessage(), pde);
+                } catch(final LockException | PermissionDeniedException e) {
```

Comment:
somehow I'd expect that more code would need to have this additional catch; clearly this is not the case. Interesting....